lib/rest/client.js

```diff
@@ -350,16 +351,16 @@ class Client {
                 endpoint: pollOptions.uri,
               }, pollCount);
 
-              resError = new Error(`Task failed: ${json1.result.result.errors}`);
+              resError = new Error(`Task failed: result.result.errors=${errors}`);
               resError.statusCode = r1.statusCode;
               resError.request = req;
               callback(resError, r1, json1);
               break;
 
             case 'SUCCESS':
-              if (json1.result.result.errors) {
-                // In this case some error occurred during the operation.
-                resError = new Error('Operation reported errors');
+              // Sometimes an error occurrs during the operation (partial failure).
+              if (json1.result.result && json1.result.result.errors) {
```

Comment:
Can't this just be `if (errors)`?

Reply:
I think so, `errors` mirrors `json1.result.result.errors`...